src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -378,8 +440,11 @@ public ShardRegion(string typeName, Func<string, Props> entityProps, ClusterShar
             _replicator = replicator;
             _majorityMinCap = majorityMinCap;
 
-            _retryTask = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(Settings.TunningParameters.RetryInterval, Settings.TunningParameters.RetryInterval, Self, Retry.Instance, Self);
             SetupCoordinatedShutdown();
+
+            _retryInterval = Settings.TuningParameters.RetryInterval;
+            _initRegistrationDelay = TimeSpan.FromMilliseconds(100).Max(new TimeSpan(_retryInterval.Ticks / 2 / 2 / 2));
```

Comment:
why `_retryInterval.Ticks / 2 / 2 / 2` ?

Reply:
this is to make the registration retry process faster at the beginning and then the retry interval is doubled until it matches the normal `_retryInterval` over 3 iterations

Reply:
I see